Source/Entities/Validation/ValidationRules.cst

```diff
@@ -164,12 +214,13 @@ namespace <%=NameSpace%>.Validation
       /// This method calls the Invoke method on each rule in the list.  If the rule fails, it 
       /// is added to the <see cref="BrokenRulesList"/>
       /// </remarks>
+	  /// <param name="target">The entity to validate the rules against.</param>
       /// <param name="ruleList">List of rules to validate.</param>
-      private void ValidateRuleList(List<ValidationRuleInfo> ruleList)
+      private void ValidateRuleList(List<ValidationRuleInfo> ruleList, IEntity target = null)

```

Comment:
default parameter values not supported in early versions of .net, need to write out overloads.

Reply:
Fixed